services/web/server/src/simcore_service_webserver/scicrunch/submodule_setup.py

```diff
@@ -2,22 +2,23 @@
     Notice that this is used as a submodule of groups'a app module
 """
 import logging
-from typing import Any, MutableMapping, Optional
 
+from aiohttp import web
 from pydantic import ValidationError
+from simcore_service_webserver.session_settings import assert_valid_config
```

Comment:
you probably need a relative import here

Reply:
Good catch!! Actually it explains the error in pylance above. 
The auto-import got the wrong import here.
I notice there are several absolute imports for local modules. I will refactor those as well